ansible/roles/test/files/ptftests/inner_hash_test.py

```diff
@@ -26,8 +26,9 @@ class InnerHashTest(BaseTest):
     #---------------------------------------------------------------------
     # Class variables
     #---------------------------------------------------------------------
-    DEFAULT_BALANCING_RANGE = 0.25
+    DEFAULT_BALANCING_RANGE = 0.3
```

Comment:
Why is this bumped up to 0.3? Typically I have found balancing range was well below 0.25 already, why do we make it less stringent?

Reply:
done